web/src/repo/settings/RepoSettingsCodeIntelligencePage.scss

```
@@ -1,8 +1,24 @@
 .repo-settings-code-intelligence-page {
     width: 100%;
+
+    &-collapsible {
+        border: 1px solid #2b3750;
```

```suggestion
        border: 1px solid $color-border;
```

nit: you can use `$color-border` https://sourcegraph.com/github.com/sourcegraph/sourcegraph@d5bb53fb4db4ecb9b1b81724df314983ad6da1ee/-/blob/shared/src/global-styles/colors.scss#L43:17